### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by olivergondza**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554</version>
+    <version>1.580</version>
```
Comment (scoheb) [inline pom.xml]: Any reason why we should not go directly to plugin 2.X and use jenkins.version to control version to build against?

Comment (olivergondza) [inline pom.xml]: I bumped this to avoid some strange but not quite investigation worthy test crash. This is just less work.

**Comment by scoheb** [comment]
+1

File: pom.xml (hunk 1 above)
Comment (rsandell) [inline pom.xml]: My guess is there would be a ton of work to adapt to the new plugin pom, at least more than necessary and should be in a separate change in that case.

File: pom.xml [hunk 2]
```diff
@@ -159,16 +149,6 @@
                         </goals>
                         <phase>compile</phase>
                     </execution>
-                    <execution>
```
Comment (rsandell) [inline pom.xml]: nah, I want my checkstyle to fail on test

**Comment by olivergondza** [comment]
There are still some failures on JDK 8.

**Comment by olivergondza** [comment]
All green, let's get this merged.

**Comment by olivergondza** [comment]
Argh, not so fast ...

**Comment by rsandell** [comment]
@olivergondza If this plugin is critical for you, you are welcome to jump on as a full maintainer. It seems like my former employer has no interest in this any more, and I don't think @TWestling would mind either.

**Comment by olivergondza** [comment]
@rsandell, so do you care for the checkstyle or not? I find nitpicking mising `@throws` Javadoc on test methods outright useless.

**Comment by rsandell** [comment]
@olivergondza not really on test classes, I find that a very strict checkstyle ruleset is good for keeping code from various sources consistent. So far I've found that fixing the nits is far more easy than tweaking the ruleset to be perfect ;)

For test code for example I usually start a file with `//CS IGNORE MagicNumber FOR NEXT 200 LINES. REASON: TestCode` instead of having separate rules for tests vs production code.

**Pull request merged**